skbeam/core/accumulators/binned_statistic.py

```diff
@@ -661,6 +661,9 @@ def __init__(self, shape, bins=10, range=None,
         mask : 2-dimensional np.ndarray of ints, optional
             array of zero/non-zero values, with shape `shape`.
             zero values will be ignored.
+        r_map : the map of pixel radii for each pixel. This is useful when the
```

Comment:
cool. to be consistent, renamed them all (`rpix` was part of original notation). also renamed `phipix` to `phi_map`.